From 1d2d6b032566060eb9791d677fc9cee4e0b368bc Mon Sep 17 00:00:00 2001
From: Mingcong Bai <jeffbai@aosc.io>
Date: Sun, 1 Dec 2024 22:56:20 +0800
Subject: [PATCH 3/3] quirks: lower AttrTrackpointMultiplier for ThinkPad
 X200s/201s to 0.25

Follow commit 65b53f82ff9f ("quirks: lower AttrTrackpointMultiplier for
ThinkPad X200/201 to 0.25") and lower this multiplier for Lenovo ThinkPad
X200s/201s models, as I found the identical issue as the one previously
quirked for ThinkPad X200/201.

Signed-off-by: Mingcong Bai <jeffbai@aosc.io>
---
 quirks/50-system-lenovo.quirks | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/quirks/50-system-lenovo.quirks b/quirks/50-system-lenovo.quirks
index 30a79fc7..404efa20 100644
--- a/quirks/50-system-lenovo.quirks
+++ b/quirks/50-system-lenovo.quirks
@@ -67,10 +67,10 @@ MatchName=*TPPS/2 IBM TrackPoint
 MatchDMIModalias=dmi:*svnLENOVO:*pvrThinkPadX20?:*
 AttrTrackpointMultiplier=0.25
 
-[Lenovo X200x Trackpoint]
+[Lenovo X200s/201s Trackpoint]
 MatchName=*TPPS/2 IBM TrackPoint
 MatchDMIModalias=dmi:*svnLENOVO:*pvrThinkPadX20??:*
-AttrTrackpointMultiplier=1.25
+AttrTrackpointMultiplier=0.25
 
 [Lenovo X230 Trackpoint]
 MatchName=*TPPS/2 IBM TrackPoint
-- 
2.47.1

